### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by gdedi001** [brennsch/prj-rev-bwfs-dasmoto#1]
_Grade: Exceeds Expectation_

Great job on your project - I loved how you were able to incorporate the `<p>` and `<h2>`  tag making your HTML very neat! In addition to this, you utilized the `<font>` tag where you incorporated the blue font color. These two aspects of your code helped you meet the design requirements! An area for improvement here is with respect to your implementation of the `class` attribute in some cases you used it incorrectly.

A recommendation here is to review the following article. Try applying the concept and its usage. It will aid you as your projects get more complex. Read it thoroughly and view the examples! Keep up the great work!
[id vs. class](https://css-tricks.com/the-difference-between-id-and-class/)

**Comment by brennsch** [comment]
Hello,

Could you be more specific about how I used the class attribute incorrectly? I used the class attribute three times and used it the exact way each time. I reviewed the article but am still confused. Do you mean that I should have used an id instead of a class attribute? 

Thank you!

**Comment by gdedi001** [comment]
Yup in your case you should have opted for `id` instead. Even though you used the class attribute 3 times, they have only been used once, and as a result, are unique. Anytime we have uniqueness go for `id`
